example/scripts/run_test.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+set -e
+
+specName=$(echo $2 | rev | cut -d'/' -f 1 | cut -d'.' -f 2 | rev)
+outputPath="$1/$specName.txt"
+
+echo $(date "+%Y%m%d-%H:%M:%S") run test $2 and output to $outputPath
+
+jasmine $2 > $outputPath
```

Comment:
Catch stderr as well.

```suggestion
jasmine $2 > 2>&1 $outputPath
```